be/src/storage/rowset/page_io.cpp

```diff
@@ -255,7 +255,8 @@ Status PageIO::read_and_decompress_page(const PageReadOptions& opts, PageHandle*
     *body = Slice(page_slice.data, page_slice.size - 4 - footer_size);
     if (opts.use_page_cache) {
         // insert this page into cache and return the cache handle
-        Status st = cache->insert(cache_key, page.get(), &cache_handle, opts.kept_in_memory);
+        ObjectCacheWriteOptions opts;
+        Status st = cache->insert(cache_key, page.get(), &cache_handle, opts);
```

Comment:
Should we set the `priority` in `ObjectCacheWriteOptions` based on the `opts.kept_in_memory`?

Reply:
Keep_in_memory already removed from opts, if needed, i will add andother item (priority) instead.